onnx/numpy_helper.py

```diff
@@ -18,11 +19,12 @@ def combine_pairs_to_complex(fa):  # type: (Sequence[int]) -> Sequence[np.comple
     return [complex(fa[i * 2], fa[i * 2 + 1]) for i in range(len(fa) // 2)]
 
 
-def to_array(tensor):  # type: (TensorProto) -> np.ndarray[Any]
+def to_array(tensor, base_dir=""):  # type: (TensorProto, Text) -> np.ndarray[Any]
```

Comment:
adding base_dir is complicating all the APIs... specially ParseData... I wonder if we should just try and read from current dir